inference/server/oasst_inference_server/routes/configs.py

```diff
@@ -36,8 +36,7 @@ class ModelInfo(pydantic.BaseModel):
     ParameterConfig(
         name="nucleus9",
         description="Nucleus sampling with p=0.9",
-        work_parameters=inference.WorkParametersInput(
-            model_name="_model_",
+        sampling_parameters=inference.SamplingParameters(
             top_p=0.9,
             temperature=0.8,
             repetition_penalty=1.2,
```

Comment:
I have seen `1/0.83` as magic-value for repetition_penalty a couple of times which is also ~1.2 .. do you know where it comes from?

Reply:
no idea, I got this from you